### DISCUSSION TITLE
Code block formatting in .md or .adoc

### DISCUSSION THREAD

**Discussion by b9Joker108** [community/community#102393] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Dear GitHub community 

I have been using Notion as an intranet, or second brain. I know, I know, it's not open source, but I am a FOSS pragmatist, not a purist. And, it is a sweet bit of code, even though proprietary. The Notion web app was starting to become really glitchy and unmanageable with the size of the page. The final straw, was when a section I had doted upon and devoted considerable time in editing, was not preserved, due to the glitch. I felt wrath and fury. 

Anyway, I channelled it appropriately, and have successfully exported, unzipped and installed the .md in its appropriate GitHub repository. For the most part, I am delighted with how it has rendered. I have already figured out how to tweak the code blocks with the the three back ticks opening and closing the code on their own lines. But, I really would like the code block background to be slightly coloured, rather than drab grey. And, the single word shell code blocks to be coloured text, with slightly tinted, coloured background. Is this possible? 

Also, the Bash syntax highlighting was not very colourful. Is that standard? Is there any way I may make Shell, Bash and Zsh syntax highlighting render polychromatically in the code block? And render the code block itself, tinted with colour? I just love colour, and it would make me so very pleased if I could realise this change.

Thanking you in anticipation
Beauford

**Comment by MasterInQuestion** [comment]
GFM (GitHub Flavored Markdown) provides the possibility to embed much HTML.
    Though ["github.com"](https://github.com/) much doesn't accept them.

    If such doesn't work:
    https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks
    ; nothing else could help.

    ----

    "Also, the Bash syntax highlighting was not very colourful."
    I noted it then.
    GitHub's syntax highlighting for "shell" (Unix Shell) appears somewhat incomplete:
    E.g. [ https://github.com/yt-dlp/yt-dlp/issues/9710#issue-2249649551](https://github.com/yt-dlp/yt-dlp/issues/9710#issue-2249649551)
<img src="https://github.com/MasterInQuestion/attach/raw/main/community/community/102393/highlight.0.webp" alt="    [ &quot;highlight.webp&quot; ]" />
    Certain tokens are not recognized (thus not highlighted).

    A probable workaround may be mistagging the syntax as "powershell": which seems to receive better tokenization yet.
    The syntax shares significant similarity with Unix Shell: though may fail some cases.
